test/integration/api/projects.js

```diff
@@ -1348,13 +1365,61 @@ describe('api: /projects?forms=true', () => {
           form.reviewStates.received.should.equal(0);
         }))));
 
+    it('should return projects with datasets', testService(async (service) => {
+      const asAlice = await service.login('alice');
+
+      await asAlice.post('/v1/projects/1/forms?publish=true')
+        .send(testData.forms.simpleEntity)
+        .set('Content-Type', 'application/xml')
+        .expect(200);
+
+      await asAlice.get('/v1/projects?datasets=true')
+        .expect(200)
+        .then(({ body }) => {
+          body[0].datasetList.length.should.equal(1);
+          body[0].datasetList[0].name.should.equal('people');
```

Comment:
This test should include a line like `datasetList[0].should.be.an.ExtendedDataset()` to look more like the forms test above: `should return projects with verbs and nested extended forms`